### PR TITLE
Endpoint Observables (A- C)

### PR DESCRIPTION
**Pull request by patel-bhavin**
Fixes yamls to ensure observable value is present in the search!

### CODE REVIEW

File: detections/endpoint/credential_dumping_via_symlink_to_shadow_copy.yml
Comment (P4T12ICK) [inline detections/endpoint/credential_dumping_via_symlink_to_shadow_copy.yml:1]: Processes.parent_process_name missing after by

File: detections/endpoint/credential_dumping_via_copy_command_from_shadow_copy.yml
Comment (P4T12ICK) [inline detections/endpoint/credential_dumping_via_copy_command_from_shadow_copy.yml:1]: Processes.parent_process_name missing after by

File: detections/endpoint/creation_of_shadow_copy_with_wmic_and_powershell.yml
Comment (P4T12ICK) [inline detections/endpoint/creation_of_shadow_copy_with_wmic_and_powershell.yml:1]: Processes.parent_process_name missing after by

File: detections/endpoint/credential_dumping_via_copy_command_from_shadow_copy.yml
Comment (patel-bhavin) [reply detections/endpoint/credential_dumping_via_copy_command_from_shadow_copy.yml:1]: updated!

**Comment by patel-bhavin** [comment]
Thank you for the comments, i updated the yamls!

**Review by mvelazc0 — APPROVED** [review]

**Pull request merged**